cmd/experimental/kjobctl/pkg/cmd/util/client_getter.go

```diff
@@ -42,7 +42,7 @@ func NewClientGetter(clientGetter genericclioptions.RESTClientGetter) ClientGett
 	}
 }
 
-func (cg *clientGetterImpl) K8sClientSet() (k8s.Interface, error) {
+func (cg *clientGetterImpl) K8sClientset() (k8s.Interface, error) {
```

Comment:
Can we do this rename in a clean-up PR?

Reply:
It's a bug. On fake we have `K8sClientset`, on impl `K8sClientSet`. In both cases I need to rename to fix it.